src/commands/flex/plugins/upgrade-plugin.ts

```diff
@@ -25,6 +25,12 @@ interface ScriptsToRemove {
   post?: string;
 }
 
+interface AddDependency {
```

Comment:
would probably be better to name this `DependencyUpdates` or `DependenciesToUpdate` or something